browser/resources/brave_new_tab_page_refresh/components/news/news_feed.tsx

```diff
@@ -30,9 +34,9 @@ export function NewsFeed() {
     {
       braveNews.isShowOnNTPPrefEnabled &&
         <div data-css-scope={style.scope}>
-          {braveNews.isOptInPrefEnabled
```

Comment:
Hey, I think this needs to be wrapped in a `React.Suspense` or its just going to block rendering

Reply:
Thanks - obviously I'm a little confused by that feature 😆